### ISSUE TITLE
Scribble / Graffiti causes OpenKore to crash

### ISSUE THREAD

**Issue by itsrachelfish** [OpenKore/openkore#329]
When a rogue uses the skill scribble it causes kore to crash. Using the skill itself shows up properly, but once the words are on the ground it causes kore to crash immediately. Logging back in while the graffiti is still on the ground will cause kore to crash again.

```
Packet Tokenizer: Unknown switch: 0A27
Player khosmita`~ (5) is casting Scribble on location (77, 55) (Delay: 0ms)

Incoming data left in the buffer:
================================================
98 bytes   Nov 24 13:28:26 2016
  0>  50 45 4E 44 45 4A 4F 00    87 00 4C A6 D8 00 00 00    PENDEJO...L.....
 16>  00 00 FF FF FF FF 9C 37    0A 0E 44 37 0A 0E E0 36    .......7..D7...6
 32>  0A 0E 00 F2 19 00 F2 55    44 00 E7 07 5B E6 80 54    .......UD...[..T
 48>  B4 00 E0 36 0A 0E 46 58    44 00 00 00 00 00 07 00    ...6..FXD.......
 64>  00 00 C0 89 1B 00 00 00    00 00 EC 41 16 77 98 F8    ...........A.w..
 80>  17 01 DC 00 1E EE 3F 00    01 00 4D 00 37 00 F1 3A    ......?...M.7..:
 96>  8E 0A                                                 ..
Disconnecting (128.241.92.101:4500)...disconnected
This program has encountered an unexpected problem. This is probably because
of a recent server update, a bug in this program, or in one of the plugins.
We apologize for this problem. You may get support from IRC or the forums.

A detailed error report has been saved to errors.txt. Before posting a bug
report, please try out the SVN version first. If you are already using the SVN
version, search the forums first to see if your problem had already been solved,
or has already been reported. If you truly believe you have encountered a bug in
the program, please include the contents of the errors.txt in your bug report,
or we may not be able to help you!

The error message is:
'X' outside of string in unpack at src/Network/Receive.pm line 2050, <GEN6> line 1.
```

**Comment by farrainbow** [comment]
Can you get more details on those?
Relevant config options:
```
debugPacket_include 099F, 09CA
debugPacket_include_dumpMethod 2
```

**Comment by AlexandreBR3N** [comment]
I recommend try to repeat the error and check the event. Reproduce the same scenario.
For example, check if the cell is walkable, what kore think about cell, what is causing the internal exception.

**Comment by itsrachelfish** [comment]
This was fixed by @allanon as of #535
